versions/3.1.0.md

```diff
@@ -75,6 +75,8 @@ Path templating refers to the usage of template expressions, delimited by curly
 
 Each template expression in the path MUST correspond to a path parameter that is included in the [Path Item](#path-item-object) itself and/or in each of the Path Item's [Operations](#operation-object).
 
+The value for these path parameters MUST NOT contain unescaped forward slashes (`/`).
```

Comment:
I believe we should also outlaw ? and # as they would change the meaning of the URL.

Reply:
We also need to tell people to refer back to RFC 3986 rules for URLs.